ecl/hql/hqlexpr.cpp

```diff
@@ -1112,7 +1112,15 @@ bool HqlParseContext::createCache(const char * simplifiedEcl, bool isMacro)
         writeStringToStream(*stream, "</Cache>\n");
         stream->flush();
     }
-    cacheFile->move(filename);
+    try
+    {
+        cacheFile->rename(filename);
```

Comment:
File move is suitable for renames across file systems and always retries on failures.  Rename designed to only works within same file system and does not attempt retries (so, it's quicker).  

Rename is more suitable here as the source & target file is in same directory and retries are unnecessary.